.github/stale.yml

```diff
@@ -0,0 +1,9 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 15
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
```

Comment:
I'm not entirely sure that it's a good idea to auto close PRs/Issues in 10 days.

Reply:
It's 10 days after they get marked as stale. So it would be 25 days total from last touch. Would 14 make you happier?

Reply:
I think 10 is MORE than enough time personally. Is there a way for admins to override?

Reply:
There's an option to ignore certain labels. We could use that as a way to override.